bot/exts/evergreen/splitify.py

```diff
@@ -0,0 +1,208 @@
+
+        7. In the for loop, as it iterates, the program multiplies single_width by width_multiplier to correctly
+        position a square (split piece) width wise. It then proceeds to paste the newly positioned square (split piece)
+        onto the new_img. The program increases the width_multiplier by 1 every iteration so the image wouldn't get
+        pasted in the same spot and the positioning would move accordingly. It makes sure to increase the
+        width_multiplier before the check, which checks if the end of a row has been reached, -
+        (index + 1) % pieces == 0, so after it, if it was True, width_multiplier would have been reset to 0 (start of
+        the row). If the check returns True, the height gets increased by a single square's (split piece) height to
+        lower the positioning height wise and, as I just mentioned, the width_multiplier gets reset to 0 and width will
+        then be calculated from the start of the new row. The for loop finishes once all the squares (split pieces) were
+        positioned accordingly.
+
+        8. Finally, it returns the new_img, the randomized squares (split pieces) stitched back into the format of the
+        original image - user's PFP.
+        """
+        random.shuffle(imgs)
+        single_img = imgs[0]
+
+        single_wdith = single_img.size[0]
+        single_height = single_img.size[1]
+
+        multiplier = int(math.sqrt(len(imgs)))
+
+        total_width = multiplier * single_wdith
+        total_height = multiplier * single_height
+
+        new_img = Image.new('RGBA', (total_width, total_height), (250, 250, 250))
+
+        width_multiplier = 0
+        height = 0
+
+        squares = math.sqrt(len(imgs))
+
+        for index, img in enumerate(imgs):
+            width = single_wdith * width_multiplier
+
+            new_img.paste(img, (width, height))
+
+            width_multiplier += 1
+
+            if (index + 1) % squares == 0:
+                width_multiplier = 0
+                height += single_height
+
+        return new_img
+
+    def splitify(self, img_bytes: BytesIO, squares: int) -> BytesIO:
+        """Seperate function run from an executor which splitifies an image."""
+        avatar = Image.open(BytesIO(img_bytes))
+        avatar = avatar.convert('RGBA').resize((1024, 1024))
+
+        img_squares = self.split_image(avatar, squares)
+        new_img = self.join_images(img_squares)
+
+        bufferedio = BytesIO()
+        new_img.save(bufferedio, format='PNG')
+        bufferedio.seek(0)
+
+        return bufferedio
+
+    @commands.command(name='splitify')
+    async def splitify_command(self, ctx: commands.Context, squares: int = 16) -> None:
+        """Splits your avatar into x squares, randomizes them and stitches them back into a new image!"""
+        async with ctx.typing():
+            if squares < 1:
+                raise commands.BadArgument('Squares must be a `positive number`:exclamation:')
```

Comment:
As mentioned, the emoji here just looks weird. We should use no exclamation at all or a ascii excalamation point, we can also drop the code formatting.
```suggestion
                raise commands.BadArgument('Squares must be a positive number')
```